.vsts/lint.yml

```diff
@@ -13,7 +13,17 @@ steps:
   name: Setup_depot_tools
 
 - bash: |
+    chromium_revision="$(grep -A1 chromium_version DEPS | tr -d '\n' | cut -d\' -f4)"
+    buildtools_revision="$(curl -sL "https://chromium.googlesource.com/chromium/src/+/${chromium_revision}/DEPS?format=TEXT" | base64 -d | grep buildtools_revision -A1 | tr -d '\n' | cut -d\' -f4)"
```

Comment:
This is wild and I love it 😆

Reply:
I will be broken in less than a month =)